sharktank/tests/models/deepseek/test_deepseek.py

```diff
@@ -4,28 +4,45 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import pytest
+import unittest
+
+import torch
 
 from sharktank.models.llm import *
 from sharktank.models.deepseek.toy_deepseek import generate
-
-import pytest
-import torch
+from sharktank.utils.load_llm import *
+from sharktank.utils.evaluate import *
 
 
-@pytest.mark.xfail(
+@pytest.mark.skip(
     reason="Deepseek support will be added soon",
 )
-def test_deepseek():
-    torch.set_default_dtype(torch.float32)
-    theta, config = generate(12345)
-    model = PagedLlmModelV1(theta=theta, config=config)
-
-    ids = torch.asarray(
-        [[3, 22, 13, 114, 90, 232, 61, 13, 244, 13, 212]], dtype=torch.int64
-    )
-    logits = model.prefill(tokens=ids)
-    ids = ids[0, :-1]
-    logits = logits[0, 1:]
-    cross_entropy = torch.nn.functional.cross_entropy(logits, ids)
-
-    assert pytest.approx(5.3062, 1e-4) == cross_entropy
+class DeepseekShardedTest(unittest.TestCase):
+    def test_deepseek(self):
+        theta, config = generate(12345)
+        model = PagedLlmModelV1(theta=theta, config=config)
+
+        ids = [[3, 22, 13, 114, 90, 232, 61, 13, 244, 13, 212]]
```

Comment:
It is suspicious that these values have not change, since on the feature branch the model config has changed so I would assume that the tokens designed to return the highest cross-entropy would have to also change.

Reply:
@archana-ramalingam

Reply:
Since the test is skipped we can move things forward and address it when we land the other changes.